src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/BaseMockSpringTest.java

```diff
@@ -136,6 +168,48 @@
     @MockBean(name = "transactionManager")
     private PlatformTransactionManager transactionManager;
 
+    @MockBean
+    protected Authentication authentication;
+    @MockBean
+    protected SecurityContext securityContext;
+    @MockBean
+    protected JwtDecoder jwtDecoder;
+
+    @Before
+    public void setUpBase() {
+        when(securityContext.getAuthentication()).thenReturn(authentication);
+        SecurityContextHolder.setContext(securityContext);
+        setSecurityAuthorities(authentication);
+        when(jwtDecoder.decode(anyString())).thenReturn(getJwt());
+    }
+
+    protected void setSecurityAuthorities(Authentication authenticationMock, String... authorities) {
+
+        Jwt jwt = getJwt();
+        when(authenticationMock.getPrincipal()).thenReturn(jwt);
+
+        Collection<? extends GrantedAuthority> authorityCollection = Stream.of(authorities)
+            .map(a -> new SimpleGrantedAuthority(a))
+            .collect(Collectors.toCollection(ArrayList::new));
+
+        when(authenticationMock.getAuthorities()).thenAnswer(invocationOnMock -> authorityCollection);
+
+    }
+
+    @NotNull
+    private Jwt getJwt() {
+        return Jwt.withTokenValue(BEARER_TOKEN)
+            .claim("exp", Instant.ofEpochSecond(1585763216))
```

Comment:
this timestamp can expire in future and test will start failing, so we need to, take current date and add some time in it

Reply:
I think only the format is being validate as we are mocking getuserinfo calls.